### DISCUSSION TITLE
HelloGitHub 徽章计划

### DISCUSSION THREAD

**Discussion by 521xueweihan** [lobehub/lobehub#3029] [General]
<img width="791" alt="image" src="https://github.com/lobehub/lobe-chat/assets/8255800/ee4d42f3-0e7e-42e9-9d20-0f5992c92177">

🎉 恭喜！您的[项目](https://hellogithub.com/repository/39701baf5a734cb894ec812248a5655a)已被 HelloGitHub 社区收录和推荐，现邀请您加入 HelloGitHub 徽章计划。加入后，您将获得以下特权：

- 社区认可: 徽章代表你的项目已通过 HelloGitHub 社区的严格筛选，并获得推荐。
- 提高曝光: 佩戴徽章后，你的项目将获得更多推荐流量，吸引更多用户和贡献者。
- 增强互动: 用户可通过徽章快速了解项目，并与之互动（点赞、评论、收藏）。
- 收集反馈: 获取广大用户的真实反馈，持续优化你的项目。
- 特殊标识: 认证后，你的评论将有显著标识，并获得置顶优先权。

📌 [立即点击](https://hellogithub.com/repository/39701baf5a734cb894ec812248a5655a/embed) 完成徽章佩戴，加入 HelloGitHub 徽章计划，让你的开源项目绽放更耀眼的光彩。

---

**HelloGitHub**，一个专注于发现、分享、推广开源项目的社区。自 2016 年创立以来，从每月一更的月刊发展到今天，已经成为一个拥有超过 **1 万** 用户的充满活力的开源社区。我们的足迹遍布多个内容平台，并获得了全网 **50 万** 粉丝的信赖与支持。

**Comment by canisminor1990** [comment]
@521xueweihan [README.zh-CN.md](https://github.com/lobehub/lobe-chat/blob/main/README.zh-CN.md) 认领失败

![image](https://github.com/lobehub/lobe-chat/assets/17870709/09ad3847-23d9-4e66-8d9c-7dcc730bc04c)

**Reply by 521xueweihan** [reply]
@canisminor1990 您好，我看了下您提交的代码，有两个问题：
1. 徽章必须添加在默认展示的 README 文件（HelloGitHub 社区正在做国际化）
2. 必须保证复制代码的完整性（因为我看了 README 的代码，如果不想混用 HTML 的话，我这边看可以修改下验证条件做一下适配）

**Reply by 521xueweihan** [reply]
@canisminor1990 感谢您的认领，已通过并全站置顶（1 天），明天会再发一条微博推荐 lobe-chat 开源项目。

<img width="778" alt="image" src="https://github.com/lobehub/lobe-chat/assets/8255800/e0b84574-dfc0-495c-94f8-0486fc774553">

**Reply by canisminor1990** [reply]
如何编辑访问中的链接，比如添加文档地址一类

**Reply by 521xueweihan** [reply]
<img width="1035" alt="image" src="https://github.com/lobehub/lobe-chat/assets/8255800/aca1d511-863c-4993-8748-f10279cb215f">

目前只能后台修改，有需要添加的地址和项目描述（错误必须修改之处），可以发给我，我立马修改！

**Reply by canisminor1990** [reply]
- 文档地址：https://lobehub.com/docs/usage/start

**Reply by 521xueweihan** [reply]
已添加文档地址，并修改了下官网地址（去掉了 /zh）

<img width="716" alt="image" src="https://github.com/lobehub/lobe-chat/assets/8255800/88065e52-533d-44d1-a06e-a1b88b9ab59a">

**Comment by 521xueweihan** [comment]
@canisminor1990 这边看到您在 README.md 文件下掉了 HelloGitHub，导致佩戴认证失败。

这边是考虑网站的内容是中文，放在 README.md 会影响用户体验吗

**Reply by canisminor1990** [reply]
是的 我们倾向于放在的对应语种的readme中

**Reply by 521xueweihan** [reply]
嗯嗯，等我完成网站的国际化，再来告知哈。

**Comment by Superblue045** [comment]
Nice to meet you

**Comment by 521xueweihan** [comment]
@canisminor1990 您好，国际化已完成[点击查看](https://hellogithub.com/en/repository/39701baf5a734cb894ec812248a5655a)。方便的话可以在 Readme 文件中添加徽章，完成认领哈。

<img width="772" alt="image" src="https://github.com/user-attachments/assets/781c7ba2-a187-483b-a946-e51b80997a21">
